INIT: Using found mnist data
INIT: Finished creating train dataset
2022-04-01 15:20:10.431178: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: UNKNOWN ERROR (34)
2022-04-01 15:20:10.431244: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (b2e041ce739c): /proc/driver/nvidia/version does not exist
2022-04-01 15:20:10.431489: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Traceback (most recent call last):
  File "RunTest.py", line 194, in <module>
    main(args)
  File "RunTest.py", line 115, in main
    print(X.shape)
AttributeError: 'tuple' object has no attribute 'shape'